### DISCUSSION TITLE
Thoughts on creating a dev discord server?

### DISCUSSION THREAD

**Discussion by dan4222** [GCCempt/GCC_OpenSong_bot#55] [Ideas]
I think we should create a consolidated discord server to test bots with multiple sets of "read" and "post" channels so that people can test their bots independently. Could also serve as a central place to discuss issues and development and to collaborate. Would also be cool to use the github discord bot so thats a bonus.

**Comment by AllDayJon** [comment]
I think this is a good idea. Bonus points if anyone wants to write a build script to create new channels to post to, and delete them when the PR closes etc- or you know, something similar.
